Projects/App/Sources/UI/Review/EvaluationViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  EvaluationViewController.swift
+//  App
+//
+//  Created by 리아 on 2022/10/20.
+//  Copyright (c) 2022 zesty. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+import UIKit
+import SnapKit
+
+final class EvaluationViewController: UIViewController {
+    
+    // MARK: - Properties
+    private let cancelBag = Set<AnyCancellable>()
+    
+    private lazy var evaluationStackView = UIStackView()
+    private lazy var goodButton = EvaluationButton(type: .good)
+    private lazy var sosoButton = EvaluationButton(type: .soso)
+    private lazy var badButton = EvaluationButton(type: .bad)
+
+    // viewModel한테서 옵셔널로 받아올 거임
+    // 사용자가 뒤로 갈 수도 있으니까
+    private var evaluation: Evaluation!
+    
+    // MARK: - LifeCycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        createLayout()
+    }
+    
+    // MARK: - Function
+    
+    @objc func goodButtonTouched() {
+        goodButton.isSelected.toggle()
+        sosoButton.isSelected = false
+        badButton.isSelected = false
+        
+        evaluation = .good
+        navigationController?.pushViewController(ReviewRegisterViewController(), animated: true)
+
+    }
+    
+    @objc func sosoButtonTouched() {
+        sosoButton.isSelected.toggle()
+        badButton.isSelected = false
+        goodButton.isSelected = false
+        
+        evaluation = .soso
+        navigationController?.pushViewController(ReviewRegisterViewController(), animated: true)
+
+    }
+    
+    @objc func badButtonTouched() {
+        badButton.isSelected.toggle()
+        goodButton.isSelected = false
+        sosoButton.isSelected = false
+        
+        evaluation = .bad
+        navigationController?.pushViewController(ReviewRegisterViewController(), animated: true)
+
+    }
+    
+}
+
+// MARK: - UI Function
+
+extension EvaluationViewController {
+    
+    private func configureUI() {
+        view.backgroundColor = .systemBackground
+        
+        evaluationStackView.axis = .horizontal
+        evaluationStackView.spacing = 10
+        evaluationStackView.distribution = .fillEqually
+        
+        goodButton.button.addTarget(self, action: #selector(goodButtonTouched), for: .touchUpInside)
+        sosoButton.button.addTarget(self, action: #selector(sosoButtonTouched), for: .touchUpInside)
+        badButton.button.addTarget(self, action: #selector(badButtonTouched), for: .touchUpInside)
+    }
+    
+    private func createLayout() {
+        view.addSubviews([evaluationStackView])
+        evaluationStackView.addArrangedSubviews([goodButton, sosoButton, badButton])
+        
+        evaluationStackView.snp.makeConstraints {
+            $0.centerX.centerY.equalToSuperview()
+            $0.horizontalEdges.equalToSuperview().inset(35)
+            $0.height.equalTo(evaluationStackView.snp.width).multipliedBy(0.45)
```

Comment:
p1;

앞에서 말한 버튼 세 개를 고정크기로 가기로 했다면 이렇게 바꿔주시고, 아니면 안바꾸셔도 됩니다!

```suggestion
            $0.centerX.centerY.equalToSuperview()
```

Reply:
2a4bb6e36b19acedf77f6b4dfb7f4de51deaf780 옙 변경 완료요!